test/parallel/test-http2-backpressure.js

```diff
@@ -46,7 +46,7 @@ function event(ee, eventName) {
   });
 }
 
-function mustCallAsync(fn, exact) {
+function mustCall(fn, exact) {
```

Comment:
Completely optional, but fwiw, I like the idea of this being named something other than `mustCall` since it _is_ something different; could also be e.g. `mustCallAndResolve` or something like that to better describe the semantics

Reply:
I get that, but it's much easier to simply reuse the name the lint rule is familiar with. If that was a common pattern, I would add an option for the lint rule to accept arbitrary function name, but I think this is the only use case, so since that's optional I'll go ahead and resolve the comment if that's ok